### PR TITLE
Add od 00 webcam notebook

### PR DESCRIPTION
**Pull request by simonzhaoms**
### Checklist:

- [ ] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by simonzhaoms** [comment]
The functions used in the 00 webcam notebook are based on 01 training notebook of @jiata.

File: detection/notebooks/00_webcam.ipynb [hunk 1]
```diff
@@ -0,0 +1,650 @@
+{
```
Comment (PatrickBue) [inline detection/notebooks/00_webcam.ipynb]: <p>From this sentence "Object detection is the canonical computer vision task of determining where a specific object, feature, or activity is in an image." remove the words ", feature, or activity".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (PatrickBue) [inline detection/notebooks/00_webcam.ipynb]: <p>Could explain here on a high level what the parameters below do. I.e. "To speed up model inference we restrict processing to use at most an image resolution of 200 pixels, and only use up to 5 candidate object proposals per image. For an information on the underlying technology see the [FAQ](FAQ.md) and the [training introduction notebook](LINK) in this directory</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (PatrickBue) [inline detection/notebooks/00_webcam.ipynb]: <ul><li>It says here that "we need to convert" but looking below there is no implicit conversion (but I guess the T.Compose(T.ToTensor)(im)) does it? That's a bit confusing. Maybe instead write "The model expects RGB images as input. Below, the RGB image is converted to a torch.FloatTensor which is then passed in to the model"</li><li>"The output of the model is a probability distribution of the <a href="https://github.com/pytorch/vision/blob/master/docs/source/models.rst#object-detection-instance-segmentation-and-person-keypoint-detection" target="_blank">labeled classes</a>" don't think that's true since for each rectangle it only outputs the label with maximum score. I think what you want to say is what classes the model was trained on and hence can detect. How about remove that sentence here, and then add in the "Load Pretrained Model" after the sentence "The <a href="https://pytorch.org/docs/stable/torchvision/models.html#object-detection-instance-segmentation-and-person-keypoint-detection" target="_blank">reported box AP</a> is 37.0 trained on the instances set of COCO train2017 and evaluated on COCO val2017. ": "Coco is a popular dataset in the research community and consists of XX images with in total XX annotated objects of XX distinct classes LINK HERE such as person, car, boat, or cup."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (PatrickBue) [inline detection/notebooks/00_webcam.ipynb]: <p>In second sentence, need to change "ImageNet" to "COCO"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

**Comment by PatrickBue** [comment]
Good first step and welcome to CVBP :)
Notebook looks good, I left a few minor comments. The PR itself however contains a lot of other files which would get erroneously merged into the jiata_OD branch and makes code review impossible. Please remove those extra files, or likely easier, create a new PR with just the 00 notebook in it (and any other file if you edited it)

File: detection/notebooks/00_webcam.ipynb (hunk 1 above)
Comment (simonzhaoms) [reply detection/notebooks/00_webcam.ipynb]: <p>Yes.  <code>T.Compose(T.ToTensor)(im))</code> carries out the conversion from (H, W, C) in [0, 255] to (C, H, W) in [0.0, 0.1].</p>

**Comment by simonzhaoms** [comment]
@PatrickBue @jiata See the new PR #297 with the only 4 files I edited.

**Comment by PatrickBue** [comment]
@simonzhaoms  Thanks for making the changes!
